components/src/core/components/Comments/Comment.vue

```diff
@@ -271,6 +255,10 @@ export default defineComponent({
       type: Boolean as PropType<boolean>,
       default: false,
     },
+    alertType: {
+      type: String,
```

Comment:
Since this is shown on deleteMode will there be a situation where we show a warning or other type of alert? I doubt it. can we remove the new prop and add hardcoded "error" type? , if a requirement comes in future we can change it @MifdhaM

Reply:
Yes. hence this is removed